apps/avatax/graphql/subscriptions/OrderConfirmed.graphql

```diff
@@ -36,6 +36,9 @@ fragment OrderConfirmedSubscription on Order {
   channel {
     id
     slug
+    taxConfiguration {
+      ...TaxConfiguration
```

Comment:
**Suggestion:** Can you get only fields that you need here? It will be much better to adds mocks if I have to use `taxCalculationStrategy` instead of mocking whole object.

Reply:
Sure. I just thought you'd be using entire tax config in `SHOPX-252` 😛

Reply:
@krzysztofzuraw fixed :)